wiki/Help_Centre/Beatmapping/en.md

```diff
@@ -0,0 +1,67 @@
+# Beatmapping (help page)
+
+*For a list of all Help Centre articles, see: [Help Centre](/wiki/Help_Centre)*
+
+## How do I upload/update my map?
+
+**You can do this through the in-game editor.**
+
+To upload your beatmap, go into the editor and hit `File` > `Upload Beatmap`.
+
+This will make your beatmap available for others to download on the website and create a dedicated thread for discussion. Updating a map is the same process for a map that has already been uploaded.
+
+Each user has a limited number of maps they can simultaneously upload. Leaving a map inactive for a month (by not updating it or not posting in its thread) will send it to the Beatmap Graveyard. To update a graveyarded beatmap, you'll need to revive it first.
+
+## The editor is telling me that the beatmap I'm trying to submit isn't mine!
```

Comment:
Might be a good idea to find a way to integrate [this article](https://osu.ppy.sh/help/wiki/Guides/BSS_Issues) here?

Reply:
Linking it with a hatnote would prob be more helpful